README.md

```diff
@@ -1,31 +1,7 @@
 # Gnosis Safe Android App [![Build Status](https://travis-ci.org/gnosis/safe-android.svg?branch=master)](https://travis-ci.org/gnosis/safe-android)
 
-**WARNING: Under development. Don't use the application with real funds! Application right now targets the Rinkeby test network. Switching to mainnet (or any other ethereum network) can be done by the user but it's its responsibility in doing so.**
-
 ## Goal
-The goal of this project is to provide a 2FA-enabled Ethereum Wallet. This can be achieved by using the new [Gnosis Safe](https://github.com/gnosis/gnosis-safe-contracts) (the new version of our [Multisig Wallet](https://github.com/gnosis/MultiSigWallet)). 
-
-You can add multiple devices to the safe and secure your wallet by requiring a set of devices to confirm each transaction. Depending on you security setting and number of devices you can still access your funds even if you loose access to a device.
-
-Deploying and interacting with your Gnosis Safe will also be much cheaper than with our Multisig Wallet.
-
-## Roadmap
-### Current Features
```

Comment:
should we leave/update list of implemented features?

Reply:
We can add this again in the future ... but this was just wrong and therefore I removed it